test/langtools/jdk/javadoc/doclet/testTypeAnnotations/typeannos/Fields.java

```diff
@@ -64,6 +67,9 @@ class ModifiedScoped {
     public final @FldA String @FldA [] @FldB [] array2Deep = null;
     public final String @FldA [] [] array2First = null;
     public final String [] @FldB [] array2Second = null;
+
+    public final @FldA int primitive = 0;
+    public final @FldA int @FldB [] primitiveArray1Deep = null;
```

Comment:
These two fields have not been verified?

Reply:
I shall remove these fields; they serve no real purpose here indeed.